Typescript errors in local build

Hello,

I encountered the following typescript errors when trying to build Datalab locally. 

```
notebook.ts(176,5): error TS2531: Object is possibly 'null'.
test/settings-manager-test.ts(58,11): error TS2721: Cannot invoke an object which is possibly 'null'.
```

I used changes in the following branch to resolve the issue:
https://github.com/parthea/datalab/commit/282f1e3087f10ce2ebb2608bea7975845e8cac7b

--- a/sources/web/datalab/polymer/notebook.ts
+++ b/sources/web/datalab/polymer/notebook.ts
@@ -172,7 +172,7 @@ async function processMessageEvent(e: MessageEvent) {
 }
 
 function sendMessageToNotebookEditor(message: IframeMessage) {
-  if (iframe) {
+  if (iframe && iframe.contentWindow) {
     iframe.contentWindow.postMessage(message, location.href);
   }
 }

--- a/sources/web/datalab/polymer/test/settings-manager-test.ts
+++ b/sources/web/datalab/polymer/test/settings-manager-test.ts
@@ -55,9 +55,8 @@ describe('SettingsManager', () => {
               'script element should have the expected src');
           documentHeadCalled = true;
           const fakeEvent = document.createEvent("Event");
-          if (script.onload)
-          {
-             script.onload(fakeEvent); // Resolve our promise
+          if (script.onload) {
+            script.onload(fakeEvent); // Resolve our promise
           }
         }
         return element;

--- a/sources/web/datalab/polymer/test/settings-manager-test.ts
+++ b/sources/web/datalab/polymer/test/settings-manager-test.ts
@@ -55,7 +55,10 @@ describe('SettingsManager', () => {
               'script element should have the expected src');
           documentHeadCalled = true;
           const fakeEvent = document.createEvent("Event");
-          script.onload(fakeEvent); // Resolve our promise
+          if (script.onload)
+          {
+             script.onload(fakeEvent); // Resolve our promise
+          }
         }
         return element;
       };

